.reuse/dep5

```diff
@@ -0,0 +1,14 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: seedvault
+Upstream-Contact: Torsten Grote <t@grobox.de>
+Source: https://github.com/seedvault-app/seedvault
+
+
+Files: *app/src/main/java/com/stevesoltys/seedvault/* *app/src/main/res/* *app/src/test/* *contactsbackup/src/*
```

Comment:
Does this mean that we don't need to add header information to all those source files?

Why is there a wildcard `*` in the beginning?

Reply:
Yes, alternatively, copyright and licensing information may be associated with a file through a DEP5 file. The intended use case of this method is large directories where including a comment header in each file is undesirable.

The dep5 file should be placed inside the `.reuse` directory

https://reuse.software/spec/#dep5